tests/update/patch_before_migration.pm

```diff
@@ -73,13 +76,46 @@ sub patching_sle {
     set_var("SYSTEM_PATCHED", 1);
 }
 
+sub install_patterns {
+    select_console 'root-console';
```

Comment:
It is already in root-console, no need to select_console

Reply:
it won't harm either - just saying